src/openrct2-ui/CursorData.cpp

```diff
@@ -650,12 +650,12 @@ namespace OpenRCT2::Ui
         &HandClosedDownCursorData,  // CURSOR_HAND_CLOSED
     };
 
-    const CursorData * CursorRepository::GetCursorData(CURSOR_ID cursorId)
+    const CursorData * CursorRepository::GetCursorData(CursorID cursorId)
```

Comment:
This function is not following our code style. Because the entire namespace is wrapped between `// clang-format off` and `// clang-format on`, this has gone unnoticed.

I propose we move the clang-format toggle, starting just before `static constexpr const CursorData BlankCursorData =` and ending just before this function.